<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cafeteria Ordering System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      
      <!-- Customer View (Page 0) -->
      <widget class="QWidget" name="customerPage">
       <layout class="QHBoxLayout" name="horizontalLayout">
        
        <!-- Left Side - Menu Selection -->
        <item>
         <widget class="QGroupBox" name="menuGroupBox">
          <property name="title">
           <string>Menu</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           
           <item>
            <widget class="QLabel" name="categoryLabel">
             <property name="text">
              <string>Select Category:</string>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           
           <item>
            <widget class="QComboBox" name="categoryComboBox">
             <property name="minimumHeight">
              <number>30</number>
             </property>
            </widget>
           </item>
           
           <item>
            <widget class="QLabel" name="itemsLabel">
             <property name="text">
              <string>Available Items:</string>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           
           <item>
            <widget class="QListWidget" name="itemsListWidget">
             <property name="minimumHeight">
              <number>300</number>
             </property>
            </widget>
           </item>
           
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="quantityLabel">
               <property name="text">
                <string>Quantity:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="quantitySpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>99</number>
               </property>
               <property name="value">
                <number>1</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           
           <item>
            <widget class="QPushButton" name="addToCartButton">
             <property name="text">
              <string>Add to Cart</string>
             </property>
             <property name="minimumHeight">
              <number>40</number>
             </property>
            </widget>
           </item>
           
          </layout>
         </widget>
        </item>
        
        <!-- Right Side - Shopping Cart -->
        <item>
         <widget class="QGroupBox" name="cartGroupBox">
          <property name="title">
           <string>Shopping Cart</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           
           <item>
            <widget class="QTextEdit" name="cartTextEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="font">
              <font>
               <family>Courier</family>
              </font>
             </property>
            </widget>
           </item>
           
           <item>
            <widget class="QPushButton" name="checkoutButton">
             <property name="text">
              <string>Checkout</string>
             </property>
             <property name="minimumHeight">
              <number>40</number>
             </property>
            </widget>
           </item>
           
           <item>
            <widget class="QPushButton" name="clearCartButton">
             <property name="text">
              <string>Clear Cart</string>
             </property>
             <property name="minimumHeight">
              <number>35</number>
             </property>
            </widget>
           </item>
           
          </layout>
         </widget>
        </item>
        
       </layout>
      </widget>
      
      <!-- Manager View (Page 1) -->
      <widget class="QWidget" name="managerPage">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        
        <item>
         <widget class="QLabel" name="managerTitleLabel">
          <property name="text">
           <string>MANAGER MODE</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="font">
           <font>
            <pointsize>16</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
        
        <item>
         <widget class="QGroupBox" name="managerGroupBox">
          <property name="title">
           <string>Menu Items Management</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           
           <item>
            <widget class="QLabel" name="managerItemsLabel">
             <property name="text">
              <string>All Menu Items:</string>
             </property>
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
            </widget>
           </item>
           
           <item>
            <widget class="QListWidget" name="managerItemsListWidget">
             <property name="minimumHeight">
              <number>300</number>
             </property>
            </widget>
           </item>
           
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             
             <item>
              <widget class="QPushButton" name="addItemButton">
               <property name="text">
                <string>Add New Item</string>
               </property>
               <property name="minimumHeight">
                <number>40</number>
               </property>
              </widget>
             </item>
             
             <item>
              <widget class="QPushButton" name="editPriceButton">
               <property name="text">
                <string>Edit Price</string>
               </property>
               <property name="minimumHeight">
                <number>40</number>
               </property>
              </widget>
             </item>
             
             <item>
              <widget class="QPushButton" name="removeItemButton">
               <property name="text">
                <string>Remove Item</string>
               </property>
               <property name="minimumHeight">
                <number>40</number>
               </property>
              </widget>
             </item>
             
            </layout>
           </item>
           
           <item>
            <widget class="QPushButton" name="saveChangesButton">
             <property name="text">
              <string>Save All Changes to File</string>
             </property>
             <property name="minimumHeight">
              <number>45</number>
             </property>
            </widget>
           </item>
           
           <item>
            <widget class="QPushButton" name="managerBackButton">
             <property name="text">
              <string>Back to Customer View</string>
             </property>
             <property name="minimumHeight">
              <number>40</number>
             </property>
            </widget>
           </item>
           
          </layout>
         </widget>
        </item>
        
       </layout>
      </widget>
      
     </widget>
    </item>
   </layout>
  </widget>
  
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  
 </widget>
 <resources/>
 <connections/>
</ui>